src/subject.ts

```diff
@@ -151,3 +225,14 @@ const parseList = (input: string): string[] => {
 
   return res.filter(item => item).map(pat => pat.trim())
 }
+
+const digestAlgorithm = (digest: string): string => {
```

Comment:
Should we also verify that the digest is of expected format (hex encoded) too, and fail early if not?

Reply:
Good idea. Added a check to ensure the digest is a hex string.